tests/services/test_CollectionProcessor.py

```diff
@@ -0,0 +1,60 @@
+import tempfile
```

Comment:
Do we want the tests in the repository directory or in the collection_manager subdirectory ? Regarding config-operator I moved everything (including tests in 'config-operator directory')

We need to discuss because collection_manager and config_operator are not organized the same way in the repostory.

Reply:
You're right, I will move /tests and /containers into /collection_manager to be consistent with how config_operator is organized.